docs/section18/Rule_18-2.md

```diff
@@ -0,0 +1,87 @@
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section:** Section 18 HVAC_SystemZoneAssignment  
+**Appendix G Section Reference:** Table G3.1.1.3  
+
+**Evaluation Context:** Each HVAC System  
+**Applicability Checks:**  
+
+1. the target string matches, which indicates:  
+	a. the space type is PUBLIC_ASSEMBLY  
+	b. climate zone CZ_0_to_3a  
+	c. area >= 120,000 ft2  
+	d. none of the G3.1.1 exceptions apply  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Evaluate whether each system serves the correct zones  
+**Data Lookup:**   
+**Function Call:** 
+
+1. get_baseline_system_types()
+2. get_zones_on_same_floor()
+3. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()
+4. baseline_system_type_compare()
+5. get_zone_target_baseline_system()
+6. get_building_total_lab_exhaust_from_zone_exhaust_fan()
+
+
+**Applicability Checks:**
+
+## Rule Logic:  
+- use the function get_baseline_system_types to get a dictionary of baseline system types and systems: `baseline_system_types_dict = get_baseline_system_types(B-RMI)`
+- use the function get_dict_of_zones_and_terminal_units_served_by_hvac_sys to determine which zones are served by each system system: `zones_and_terminal_unit_list_dict = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`
+- use the function get_zone_target_baseline_system to get the target baseline system types for each zone.  This will be used when checking for laboratory systems: `target_baseline_systems = get_zone_target_baseline_system(P_RMI, B_RMI)`
+- use the function get_building_total_lab_exhaust_from_zone_exhaust_fan to get the totat lab exhaust air flow from zone exhaust fans.  This will be used when checking for laboratory systems: `lab_zone_exhaust = get_building_total_lab_exhaust_from_zone_exhaust_fan(P_RMI)`
+- create a list of laboratory zones meeting G3_1_1d: `laboratory_zones_meeting_G3_1_1d = [x for x in target_baseline_systems if target_baseline_systems[x]["SYSTEM_ORIGIN"] == G3_1_1d]`
+- evaluate the rule for each HVAC system in the B_RMI: `for hvac_system in B_RMI...heating_ventilating_air_conditioning_systems:`
+	- set the result to FAIL: `result = FAIL`
+	- find the hvac_system type by looking through the baseline_system_types_dict for the hvac_system.id: `for hvac_system_type in baseline_system_types_dict:`
+		- find the actual system type of the system (for example, SYS_5 instead of SYS_5B).  Start by looping through the possible system types: `for possible_type in [SYS_1,SYS_2,SYS_3,SYS_4,SYS_5,SYS_6,SYS_7,SYS_8,SYS_9,SYS_10,SYS_11,SYS_12,SYS_13):`
+			- set system_type to possible_type if the result of using the baseline_system_type_compare function is TRUE: `if baseline_system_type_compare(hvac_system_type,possible_type,FALSE): system_type = possible_type`
```

Comment:
Why do we need to know the actual system type?

Reply:
to check for duplicate systems of the same type on the same floor

Reply:
get_baseline_system_type returns system types including letters (5a, 5b, etc.) So why do we need to look into "possible_type"? Then baseline_system_type_compare(TRUE) can be called.

Reply:
discussed on call 7/20/2023, keep as is